scripts/globals/items/tube_of_clear_salve_i.lua

```diff
@@ -1,44 +1,51 @@
 -----------------------------------------
 -- ID: 5837
 -- Item: tube_of_clear_salve_i
--- Item Effect: Instantly removes one negative status effect from pet
+-- Item Effect: Instantly removes 1-2 negative status effects at random from pet
 -----------------------------------------
 require("scripts/globals/settings")
 require("scripts/globals/msg")
 
 function onItemCheck(target)
-    if (not target:hasPet()) then
+    if not target:hasPet() then
         return tpz.msg.basic.REQUIRES_A_PET
     end
     return 0
 end
 
 function onItemUse(target)
     local pet = target:getPet()
-    
+    local effects = {
```

Comment:
```lua
    local effects =
    {
        tpz.effect.PETRIFICATION,
        tpz.effect.SILENCE,
        tpz.effect.BANE,
        tpz.effect.CURSE_II,
        tpz.effect.CURSE_I,
        tpz.effect.PARALYSIS,
        tpz.effect.PLAGUE,
        tpz.effect.POISON,
        tpz.effect.DISEASE,
        tpz.effect.BLINDNESS
    }
```
to conform to the [style guide](https://github.com/project-topaz/topaz/blob/release/CONTRIBUTING.md#allman-braces)